utils/smartpaste/mattermost.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Options} from 'turndown';
```

Comment:
Do we need any special rules for elements that are replaced by `messageHtmlToComponent` like emojis and at-mentions?

Reply:
No, they are automatically covered (I already tested it)